deployer/scripts/upgrade.sh

```diff
@@ -513,6 +514,14 @@ function regenerate_secrets_and_support_objects() {
   generate_support_objects
 }
 
+function remove_triggers() {
+
+  for dc in $(oc get dc -l logging-infra -o jsonpath='{.items[?(@.spec.triggers[*].type)].metadata.name}'); do
```

Comment:
ok i see, but i'm not sure that we care about unnecessary patching when it's idempotent and doesn't really have any immediate effect.